keyboards/gkeyboard/greatpad/info.json

```diff
@@ -0,0 +1,158 @@
+{
+    "manufacturer": "gkeyboard",
+    "keyboard_name": "GreatPad",
+    "url": "https://github.com/gkeyboard/GreatPad",
+    "maintainer": "gkeyboard",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "device_version": "1.0.0",
+        "vid": "0x474B",
+        "pid": "0x4203"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "oled": true
+    },
+    "matrix_pins": {
+        "cols": ["GP24", "GP23", "GP16", "GP11", "GP10"],
+        "rows": ["GP6", "GP5", "GP4", "GP3", "GP2"]
+    },
+    "diode_direction": "COL2ROW",
+    "dynamic_keymap": {
+        "layer_count": 10
+    },
+    "encoder": {
+        "enabled": true,
+        "rotary": [
+            {"pin_a": "GP26", "pin_b": "GP25"},
+            {"pin_a": "GP1", "pin_b": "GP0"}
+        ]
```

Comment:
```suggestion
        "rotary": [
            {"pin_a": "GP25", "pin_b": "GP26"},
            {"pin_a": "GP0", "pin_b": "GP1"}
        ]
```

Reply:
Rotary Encoder  not work if delete "enable":true,